cmd/ci-secret-bootstrap/main.go

```diff
@@ -502,13 +502,14 @@ func updateSecrets(getters map[string]Getter, secretsMap map[string][]*coreapi.S
 					shouldCreate = true
 				}
 
-				if !shouldCreate {
-					for k, v := range existingSecret.Data {
-						if _, exists := secret.Data[k]; exists {
-							continue
-						}
-						secret.Data[k] = v
+				for k := range existingSecret.Data {
```

Comment:
If we just add logging in this PR, we should be able to not change this code, right?

Reply:
We change it in a way that we stop keeping the keys that already exist in the existing secret but not in the new one.